frontend/src/components/opportunity/ZipDownloadButton.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { useTranslations } from "next-intl";
+import { Button } from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+type ZipDownloadProps = {
+  opportunityId: number;
+};
+
+const downloadAttachmentsZip = (opportunityId: number) => {
+  const path = `/api/attachment-download/${opportunityId}`;
```

Comment:
I think this needs to be adjusted to match the route

```suggestion
  const path = `/api/opportunities/${opportunityId}/attachment-download`;
```